src/clincoded/static/components/curator.js

```diff
@@ -370,6 +370,43 @@ var RecordHeader = module.exports.RecordHeader = createReactClass({
         );
     },
 
+    /**
+     * Method to display publication status of classification in gene-disease record header
+     * @param {array} snapshots - List of snapshots associated with classification
+     */
+    renderPublishStatus(snapshots) {
+        const sortedSnapshots = snapshots && snapshots.length ? sortListByDate(snapshots, 'date_created') : [];
+        let publishedSnapshot = null;
+        let publishedWarningMessage = null;
+
+        for (let i = 0; i < sortedSnapshots.length; i++) {
```

Comment:
@bryanwulf, how about changing this line to `for (let snapshot of sortedSnapshots) {...}`?